system/tests/kohana/MinionTaskTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Tests the Minion library
+ *
+ * @group kohana
+ * @group kohana.core
+ * @group kohana.core.config
+ *
+ * @package    Koseven
+ * @category   Tests
+ * @author     Koseven Team
+ * @author     Piotr Gołasz <pgolasz@gmail.com>
+ * @copyright  (c) Kohana Team
+ * @license    https://koseven.ga/LICENSE.md
+ */
+class MinionTaskTest extends Unittest_TestCase
+{
+
+	/**
+	 * Tests that Minion Task Help works assuming all other tasks work aswell
+	 */
+	public function test_minion_runnable()
+	{
+		$minionResponse = Minion_Task::factory(['task' => 'help']);
```

Comment:
Do I understand correctly that this test simply calls a task named "help"?

The fact is that the task will be executed regardless of whether the correct path is specified in the DOCROOT.

Reply:
In unittest bootstrap i included autoloader.php (https://github.com/koseven/koseven/pull/194/files#diff-d97290f5a2964424212543942a6ede21R65)

Therefore if Minion task has executed correctly, that would mean that the docroot path is also correct.

Reply:
Thanks you!